docs/templating/twig_function_reference/user_twig_functions.md

```diff
@@ -0,0 +1,43 @@
+---
+description: User Twig functions enable getting repository user information.
+page_type: reference
+---
+
+# User Twig functions
+
+### `ibexa_user_get_current()`
+
+`ibexa_user_get_current()` returns the User object (`Ibexa\Contracts\Core\Repository\Values\User\User`) of the current user.
+
+``` html+twig
+{{ ibexa_user_get_current().login }}
+```
+
+You can get the underlying Content item, for example the user's name,
+by accessing the `content` property:
+
+``` html+twig
+{{ ibexa_render(ibexa_user_get_current().content) }}
+```
```

Comment:
This example is incorrect.

Reply:
@adamwojs 
The example was merely moved from a different place in the docs.
Please look [here](https://doc.ibexa.co/en/latest/templating/twig_function_reference/other_twig_filters/). 
If the original example was wrong, please help me with a new one.

BTW, what is the difference between `ibexa_user_get_current()` and `ibexa_get_current_user()`?

Reply:
The actual example is risky. On a clean install, while using Anonymous, it ends with this error:
`An exception has been thrown during the rendering of a template ("The User does not have the 'read|view_embed' 'content' permission with: contentId '10', locationId 'n/a'").` (`ibexa_render` default view type is [`embed`](https://github.com/ibexa/core/blob/v4.6.0-beta3/src/lib/MVC/Symfony/Templating/RenderContentStrategy.php#L21)).

The `embed` view is a link to the Content item, the link's text is the Content item's name. So, it does a bit more than the example introduction says.

To just display the name, `{{ ibexa_content_name(ibexa_user_get_current()) }}` is enough and can be used with Anonymous.

But to stick to the introduction, I would propose an example not displaying anything but accessing as promised:
`{% set current_user_name = ibexa_user_get_current().content.name %}`

Reply:
`{{ ibexa_render_field(ibexa_user_get_current().content, 'last_name') }}` looks good to me.

Reply:
`{{ dump(ibexa_user_get_current().fields, ibexa_user_get_current().content.fields) }}` shows a difference between the two. But I do not find an example which would stay on a recommandable path…

Maybe we could avoid providing a working but ugly example and just say:

```markdown
You can get the underlying Content item by accessing the `content` property: `ibexa_user_get_current().content`
```

Or consider that specifying `.content` is a best practice over letting some automatic casting to happen:

---

You can get the underlying Content item, for example to display the user's last name,
by accessing the `content` property:

``` html+twig
{{ ibexa_render_field(ibexa_user_get_current().content, 'last_name') }}
```

---